File: URI Programs/C/1020.c
#include <stdio.h>

int main(){
	int day, year = 0, month = 0;
	scanf("%d", &day);
	while(day >= 365){
		day = day - 365;
		year = year + 1;
	}
	while(day >= 30){
		day = day - 30;
		month = month + 1;
	}
	printf("%d ano(s)\n", year);
	printf("%d mes(es)\n", month);
	printf("%d dia(s)\n", day);
	return 0;
}

File: URI Programs/Python/1020.py
x1 = int(input())

year = (x1)/365
year = int(year)
x2 = x1 - year*365

month = (x2)/30 
month = int(month)
x3 = x2 - month*30
day = x3

print(str(year)+" ano(s)")
print(str(month)+" mes(es)")
print(str(day)+" dia(s)")